sharkfuser/tests/CMakeLists.txt

```diff
@@ -25,6 +25,21 @@ if(NOT SHARKFUSER_EXTERNAL_FILECHECK)
   endif()
 endif()
 message(STATUS "Using filecheck: ${SHARKFUSER_EXTERNAL_FILECHECK}")
+# wrap FileCheck in a CMake target interface
+add_executable(FileCheck IMPORTED GLOBAL)
```

Comment:
Let's use `filecheck` (all-lower-case) to disambiguate it from the LLVM native `FileCheck`.